### PR TITLE
Implement Python 3.10 layers

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

**Comment by gliptak** [comment]
@newrelic please review

File: python/publish-layers.sh [hunk 1]
```diff
@@ -90,51 +92,51 @@ function publish-python38-x86 {
     done
 }
 
-function build-python39-arm64 {
```
Comment (umaannamalai) [inline python/publish-layers.sh:93]: It looks like the python39 build functions were replaced by the new python310. Can you add back the python39 ones as well?

**Review by umaannamalai — CHANGES_REQUESTED** [review]

File: python/publish-layers.sh (hunk 1 above)
Comment (gliptak) [reply python/publish-layers.sh]: copy pasting. corrected. thank you

Comment (gliptak) [reply python/publish-layers.sh]: @umaannamalai any other updates needed?

Comment (umaannamalai) [reply python/publish-layers.sh]: Looks good - we will merge. Thanks for contributing!

**Review by umaannamalai — APPROVED** [review]

**Comment by gliptak** [comment]
@umaannamalai would you share a timeline for releasing new layers?

**Comment by gliptak** [comment]
@newrelic would you share a timeline for releasing new layers?

https://layers.newrelic-external.com/

**Comment by umaannamalai** [comment]
Hi @gliptak - we are working on our end to coordinate a release to go out in the next week or two.

**Comment by umaannamalai** [comment]
@gliptak Python 3.10 layers should be available now: https://layers.newrelic-external.com/. Please let us know if you run into any issues.

**Pull request merged**